docs/docs/porting-create-react-app-to-gatsby.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Porting Create React App to Gatsby
```

Comment:
I think `Porting from Create React App to Gatsby` would be a little more precise! Since we're talking about sites built with those tools, not the tools themselves.